test/data/manifests/fedora_34-x86_64-vhd-boot.json

```diff
@@ -8833,17 +11377,17 @@
   "image-info": {
     "/etc/resolv.conf": [],
     "boot-environment": {
-      "kernelopts": "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac ro biosdevname=0 rootdelay=300 console=ttyS0 earlyprintk=ttyS0 net.ifnames=0"
+      "saved_entry": "ffffffffffffffffffffffffffffffff-5.15.13-100.fc34.x86_64"
     },
     "bootloader": "grub",
     "bootmenu": [
       {
         "grub_arg": "--unrestricted",
         "grub_class": "fedora",
         "grub_users": "$grub_users",
-        "initrd": "/boot/initramfs-5.15.13-100.fc34.x86_64.img",
-        "linux": "/boot/vmlinuz-5.15.13-100.fc34.x86_64",
-        "options": "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac ro no_timer_check net.ifnames=0 console=tty1 console=ttyS0,115200n8",
```

Comment:
this one is also suspicious, but probably OK. Mainly since there was previously
```
"kernelopts": "root=UUID=76a22bf4-f153-4541-b6c7-0332c0dfaeac ro biosdevname=0 rootdelay=300 console=ttyS0 earlyprintk=ttyS0 net.ifnames=0"
```
which didn't get reflected in the `"options"`. Now it does.... 🤷